### PR TITLE
Update FAQ to make more sense after tensor/variable merge

### PR DESCRIPTION
**Pull request by goldsborough**
Looks like there's already a note about the `total_loss += loss` trouble. I've updated it to make more sense after the tensor/variable merge.

Also, GitHub doesn't allow pull requests for the wiki. Can someone please edit https://github.com/pytorch/pytorch/wiki/Breaking-Changes-from-Variable-and-Tensor-merge with something like

```
## Possible Issues

* If you have code that accumulates losses across iterations (e.g. to compute an
average at the end of an epoch), such as `total_loss += loss` where `loss` is
your per-iteration loss, you may find increased memory usage in your program.
This is because `loss` probably used to be a Python float (such as when it is
the result of `.sum()`), while it is now a zero-dim Tensor. `total_loss` is
thus accumulating `Tensor`s and their gradient history, which may keep around
large autograd graphs for much longer than necessary. To fix this, be sure to
convert the per-iteration loss to a Python number as soon as possible, for
example with `total_loss += float(loss)`.
```

@ezyang @apaszke @soumith

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
wiki applied

**Review by apaszke — APPROVED** [review]
There's also `loss.item()`, but I think `float(loss)` is ok too
